### PR TITLE
fix: LPS-89596 Changed position of drag icon for IE11

### PR DESCRIPTION
**Pull request by julien**

### CODE REVIEW

File: ckeditor/ckeditor.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
 For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
 */
-(function(){window.CKEDITOR&&window.CKEDITOR.dom||(window.CKEDITOR||(window.CKEDITOR=function(){var a=/(^|.*[\\\/])ckeditor\.js(?:\?.*|;.*)?$/i,g={timestamp:"J34B",version:"4.11.3",revision:"8b53603e8",rnd:Math.floor(900*Math.random())+100,_:{pending:[],basePathSrcPattern:a},status:"unloaded",basePath:function(){var c=window.CKEDITOR_BASEPATH||"";if(!c)for(var e=document.getElementsByTagName("script"),b=0;b<e.length;b++){var d=e[b].src.match(a);if(d){c=d[1];break}}-1==c.indexOf(":/")&&"//"!=c.slice(0,
+(function(){window.CKEDITOR&&window.CKEDITOR.dom||(window.CKEDITOR||(window.CKEDITOR=function(){var a=/(^|.*[\\\/])ckeditor\.js(?:\?.*|;.*)?$/i,g={timestamp:"J4E8",version:"4.11.3 DEV",revision:"c77aca6fd",rnd:Math.floor(900*Math.random())+100,_:{pending:[],basePathSrcPattern:a},status:"unloaded",basePath:function(){var c=window.CKEDITOR_BASEPATH||"";if(!c)for(var e=document.getElementsByTagName("script"),b=0;b<e.length;b++){var d=e[b].src.match(a);if(d){c=d[1];break}}-1==c.indexOf(":/")&&"//"!=c.slice(0,
```
Comment (wincent) [inline ckeditor/ckeditor.js]: So this SHA (c77aca6fd) is a non-public SHA. Not sure if it is worth trying to stop it from appearing here (probably not, for now).

**Comment by julien** [comment]
> But what do you think of?
> 
> ```
> fix: LPS-89596 Changed position of drag icon for IE11
> chore: make build of v4.11.3 plus patches
> ```
> 
> Or something like that?
> 
> Reason I am thinking this is that we might end up adding more than one patch between updates, at which point the final build is just the chore of making a new artifact for use in liferay-portal:
> 
> ```
> fix: LPS-12345 Fix an IE bug
> fix: LPS-12346 Fix another IE bug
> fix: LPS-12345 Fix yet another IE bug
> chore: update submodule to upstream v4.11.4
> chore: make build of v4.11.4 plus patches
> ```
> 
> etc.

Yes that sounds like a better idea, I'll update this PR, Thanks!

**Comment by julien** [comment]
@wincent I might have not understood something correctly but we can review this together

**Comment by julien** [comment]
While I'm at it, I'm going to take advantage of this _awesome_ script to update to the latest tag in `ckeditor-dev` (4.11.4 at this time) and apply that patch on top.

**Pull request merged**